**Core -> Full Integration (with SecurityManager)** [Build 15523](https://ci.wildfly.org/buildConfiguration/WildFlyCore_PullRequest_WildFlyCoreFullIntegrationBootableJarJdk21/568857) outcome was **FAILURE** using a merge of 17a774f282f28016ece8ff7264fa9fac2c321aa3
Summary: Tests failed: 1 (1 new), passed: 4286, ignored: 49 Build time: 02:43:55
### Failed tests
```
org.wildfly.test.integration.microprofile.reactive.messaging.kafka.api.ReactiveMessagingKafkaUserApiTestCase: org.wildfly.test.integration.microprofile.reactive.messaging.kafka.api.ReactiveMessagingKafkaUserApiTestCase.: org.testcontainers.containers.ContainerLaunchException: Container startup failed for image apache/kafka-native:3.8.0
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:346)
	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:317)
	at org.wildfly.test.integration.microprofile.reactive.RunKafkaSetupTask.setup(RunKafkaSetupTask.java:40)
	at org.jboss.as.arquillian.container.ServerSetupObserver$ServerSetupTaskHolder.setup(ServerSetupObserver.java:259)
	at org.jboss.as.arquillian.container.ServerSetupObserver.executeSetup(ServerSetupObserver.java:186)
	at org.jboss.as.arquillian.container.ServerSetupObserver.handleBeforeDeployment(ServerSetupObserver.java:119)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:331)
	... 87 more
Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container
	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:551)
	at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:341)
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)
	... 88 more
Caused by: java.lang.IllegalStateException: Wait strategy failed. Container exited with code 99
	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:520)
	... 90 more
Caused by: org.testcontainers.containers.ContainerLaunchException: Timed out waiting for log output matching '.*Transitioning from RECOVERY to RUNNING.*'
	at org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy.waitUntilReady(LogMessageWaitStrategy.java:47)
	at org.testcontainers.containers.wait.strategy.AbstractWaitStrategy.waitUntilReady(AbstractWaitStrategy.java:52)
	at org.testcontainers.containers.GenericContainer.waitUntilContainerStarted(GenericContainer.java:904)
	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:487)
	... 90 more
------- Stdout: -------
 [0m15:24:18,042 INFO  [org.jboss.modules] (main) JBoss Modules version 2.1.6.Final
 [0m [0m15:24:18,111 INFO  [org.wildfly.jar] (main) WFLYJAR0007: Installed server and application in /opt/buildAgent/work/e8e0dd9c7c4ba60/full/testsuite/integration/expansion/target/wildfly, took 359ms
 [0m [0m15:24:18,675 INFO  [org.wildfly.jar] (main) WFLYJAR0008: Server options: [-Dts.wildfly.version=41.0.0.Beta1-SNAPSHOT, --read-only-server-config=standalone.xml]
 [0m [0m15:24:18,780 INFO  [org.jboss.msc] (main) JBoss MSC version 1.5.6.Final
 [0m [0m15:24:18,985 INFO  [org.jboss.as] (MSC service thread 1-2) WFLYSRV0049: WildFly 41.0.0.Beta1-SNAPSHOT (WildFly Core 33.0.0.Beta2) starting
 [0m [0m15:24:21,047 INFO  [org.wildfly.security] (Controller Boot Thread) ELY00001: WildFly Elytron version 2.9.1.Final
 [0m [0m15:24:22,388 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0039: Creating http management service using socket-binding (management-http)
 [0m [0m15:24:22,468 INFO  [org.xnio] (MSC service thread 1-6) XNIO version 3.8.16.Final
 [0m [0m15:24:22,491 INFO  [org.xnio.nio] (MSC service thread 1-6) XNIO NIO Implementation Version 3.8.16.Final
 [0m [0m15:24:22,641 INFO  [org.jboss.remoting] (MSC service thread 1-9) JBoss Remoting version 5.0.31.Final
 [0m [0m15:24:22,727 INFO  [org.jboss.as.connector.subsystems.datasources] (ServerService Thread Pool -- 45) WFLYJCA0004: Deploying JDBC-compliant driver class org.h2.Driver (version 2.4)
 [0m [0m15:24:22,803 INFO  [org.wildfly.extension.health] (ServerService Thread Pool -- 53) WFLYHEALTH0001: Activating Base Health Subsystem
 [0m [0m15:24:22,822 INFO  [org.wildfly.extension.io] (ServerService Thread Pool -- 55) WFLYIO001: Worker 'default' has auto-configured to 8 IO threads with 64 max task threads based on your 4 available processors
 [0m [0m15:24:22,839 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 54) WFLYCLINF0001: Activating Infinispan subsystem.
 [0m [0m15:24:22,871 INFO  [org.jboss.as.jaxrs] (ServerService Thread Pool -- 56) WFLYRS0016: RESTEasy version 7.0.2.Final
 [0m [0m15:24:23,052 INFO  [org.wildfly.extension.microprofile.health.smallrye] (ServerService Thread Pool -- 65) WFLYMPHEALTH0001: Activating MicroProfile Health Subsystem


```

--- a/testsuite/standalone/src/test/java/org/jboss/as/test/integration/logging/syslog/SyslogHandlerTestCase.java
+++ b/testsuite/standalone/src/test/java/org/jboss/as/test/integration/logging/syslog/SyslogHandlerTestCase.java
@@ -69,7 +69,8 @@ public class SyslogHandlerTestCase extends AbstractLoggingTestCase {
     private static final ModelNode JSON_FORMATTER_ADDR = createAddress("logging-profile", "syslog-profile", "json-formatter", "JSON");
     private static final ModelNode SYSLOG_PROFILE_ADDR = createAddress("logging-profile", "syslog-profile");
     private static final ModelNode SYSLOG_HANDLER_ADDR = createAddress("logging-profile", "syslog-profile", "syslog-handler", "SYSLOG");
-    private static final ModelNode SYSLOG_PROFILE_ROOT_LOGGER_ADDR = createAddress("logging-profile", "syslog-profile", "root-logger", "ROOT");
+    private static final ModelNode SYSLOG_PROFILE_LOGGER_ADDR = createAddress("logging-profile", "syslog-profile", "logger",
+            LoggingServiceActivator.LOGGER.getName());
 
     /**
      * Syslog server port.
@@ -250,7 +251,7 @@ public class SyslogHandlerTestCase extends AbstractLoggingTestCase {
             op.get("enabled").set("true");
             builder.addStep(op);
 
-            op = Operations.createAddOperation(SYSLOG_PROFILE_ROOT_LOGGER_ADDR);
+            op = Operations.createAddOperation(SYSLOG_PROFILE_LOGGER_ADDR);
             op.get("level").set("TRACE");
             op.get("handlers").add("SYSLOG");
             builder.addStep(op);

